librabbitmq/amqp_openssl.c

```diff
@@ -30,7 +30,7 @@
 static int initialize_ssl_and_increment_connections(void);
 static int decrement_ssl_connections(void);
 
-static unsigned long ssl_threadid_callback(void);
+static pthread_t ssl_threadid_callback(void);
```

Comment:
Something about this declaration is causing the win32 build to fail (see checks).

Please resolve that issue.